it/it-tests/src/test/java/it/dbCleaner/PurgeTest.java

```diff
@@ -178,6 +179,7 @@ public void should_keep_only_one_snapshot_per_day() {
    * SONAR-7175
    */
   @Test
+  @Ignore("Should be reactivated when SONAR-7305 is implemented")
```

Comment:
Indeed, that was the goal of this commit, I'll do some clean up as soon as your PR is merged